importers/houzz.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/zxsecurity/steamer/importers/util"
+)
+
+type TemplateLineParser struct{}
+
+func (t TemplateLineParser) ParseLine(line string) ([]interface{}, error) {
+	entries := make([]interface{}, 0)
+	// split a line in the text file
+	data := util.SplitString(line, '\s', true, true)
+	// check data here
+	if len(data) > 2 || strings.Contains(data[3], "SEC"){
```

Comment:
I think your logic here is inverted. I think you want:
```
if len(data) < 5 || !strings.Contains(data[3], "SEC") {
```

Did you test that this importer works correctly?

Reply:
Ah yup that's my bad, added those checks after I tested it and didn't retest. I'll fix it up now